games/o.yaml

```
@@ -1422,6 +1422,28 @@
   updated: 2015-04-15
   url: http://identicalsoftware.com/ostrichriders/
 
+- name: Outpost HD
+  type: remake
+  originals:
+  - Outpost
+  repo: https://github.com/ldicker83/OPHD
+  url: https://forum.outpost2.net/index.php/topic,5718.0.html
+  development: sporadic
+  status: playable
+  license:
+  - BSD
+  content: free
```

```suggestion
  content: commercial
```
Looks like they haven't replaced the original assets yet. I'm not sure if the game is freeware yet, so this should be commercial